radio/src/targets/taranis/CMakeLists.txt

```diff
@@ -173,7 +179,7 @@ elseif(PCB STREQUAL X7)
     set(CLI ON CACHE BOOL "Enable CLI")
   elseif(PCBREV STREQUAL TLITEF4)
     set(DEFAULT_INTERNAL_MODULE MULTIMODULE CACHE STRING "Default internal module")
-    set(INTERNAL_MODULE_SERIAL YES)
```

Comment:
Was `INTERNAL_MODULE_SERIAL` supposed to be knocked off here?

Reply:
No sir ! (thx for the catch)